plugins/background_fetch/background_fetch.cc

```diff
@@ -552,28 +569,22 @@ TSPluginInit(int argc, const char *argv[])
 
   TSCont cont = TSContCreate(cont_handle_response, nullptr);
 
-  gConfig = new BgFetchConfig(cont);
-
-  while (true) {
-    int opt = getopt_long(argc, (char *const *)argv, "lc", longopt, nullptr);
-
-    switch (opt) {
-    case 'l':
-      BgFetchState::getInstance().createLog(optarg);
-      break;
-    case 'c':
-      TSDebug(PLUGIN_NAME, "config file '%s'", optarg);
-      gConfig->readConfig(optarg);
-      break;
-    }
+  if (!gConfig) {
```

Comment:
Is this checked because of potential remap use? Would it be reasonable to assume that if `TSPluginInit` is called at all, it's called before any remap rules?

Reply:
It’s a remnant from something stupid, I’ll remove it.